pkg/services/sqlstore/migrations/ualert/alert_rule.go

```diff
@@ -188,8 +188,6 @@ func transNoData(s string) (string, error) {
 		return "NoData", nil
 	case "alerting":
 		return "Alerting", nil
-	case "keep_state":
```

Comment:
This and the follows from Exec Error are still valid entries from the source (dashboard alerts). So if we remove them it will error.

Instead we should add "keep_state" to the above case statement and return "Alerting" if it is the input.